### ISSUE TITLE
Not working on windows

### ISSUE THREAD

**Issue by dungeon2567** [amrbashir/tauri-plugin-single-instance#2]
This plugin is not working on windows. I can still open multiple instances of tauri app

**Comment by dungeon2567** [comment]
![image](https://user-images.githubusercontent.com/31074903/177428224-5fa2d5c3-610d-49c0-ace9-0d0a31862c4f.png)
This is my main.rs, maybe it does not work with tray or menu?

**Comment by amrbashir** [comment]
I don't think so. So please try to provide a minimal repo and I will investigate it further.
